### ISSUE TITLE
Tomcat and Dubbo Monitoring

### ISSUE THREAD

**Issue by weenyaimac** [apache/incubator-skywalking#996]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [] Question or discussion

___
### Question
- What do you want to know?

We currently have a project like this: the Web tier is tomcat 8.5.x+spring mvc 4.3.x, and the Server tier is tomcat 8.5.x+dubbo 2.8.4, and I deploy the single version of skywalking 3.2.6 according to the quick start, which can be monitored. But there are two problems:
1) In topology, the web trace and the server trace are separated, that is, two branches, can it be merged into one branch, and I've deployed the dubbo monitor.
2) The information such as JVM and CPU is not monitored in UI, and there isn't JVM monitoring item in the Menu in Dashboard.

**Comment by wu-sheng** [comment]
Please rewrite the issue description in English for helping global community to find existed issue.

**Comment by weenyaimac** [comment]
We currently have a project like this: the Web tier is tomcat 8.5.x+spring mvc 4.3.x, and the Server tier is tomcat 8.5.x+dubbo 2.8.4, and I deploy the single version of skywalking 3.2.6 according to the quick start, which can be monitored. But there are two problems:
1) In topology, the web trace and the server trace are separated, that is, two branches, can it be merged into one branch, and I've deployed the dubbo monitor.
2) The information such as JVM and CPU is not monitored in UI, and there isn't JVM monitoring item in the Menu in Dashboard.

**Comment by wu-sheng** [comment]
@weenyaimac Updated your issue, and removed your latest comments.

**Comment by wu-sheng** [comment]
> In topology, the web trace and the server trace are separated, that is, two branches, can it be merged into one branch, and I've deployed the dubbo monitor.

Are you active `com.alibaba.dubbo.monitor.support.MonitorFilter` filter in Dubbo? Our instrumentation based on that.

> The information such as JVM and CPU is not monitored in UI, and there isn't JVM monitoring item in the Menu in Dashboard.

Which version do you use for tests? If you have any data, JVM definitely metrics are definite included.

**Comment by weenyaimac** [comment]
@wu-sheng Thanks first. I remember MonitorFilter in Dubbo is activeted by default. Pls see following dubbo code:
`@Activate(group = { **Constants.PROVIDER, Constants.CONSUMER** }) `
`public class MonitorFilter implements Filter {}`

I just solved the first question, in the way of which I deleted the tomcat plugin of the agent in the service tier. As of the second question, I test my project using v3.2.6 release. Do the two applications need to open JMX or configure application.yml of collector?
